packages/eventual-send/src/index.js

```diff
@@ -224,13 +236,24 @@ export function makeHandledPromise(Promise) {
     return handledP;
   }
 
-  HandledPromise.prototype = Promise.prototype;
+  HandledPromise.prototype = promiseProto;
 
   // Uncomment this line if needed for conformance to the proposal.
   // Currently the proposal does not specify this, but we might change
   // our mind.
   // Object.setPrototypeOf(HandledPromise, Promise);
```

Comment:
We change our mind. We must have this. Otherwise, a promise is detectably different from a handled promise, which is not the intention.

We need to update the proposal to match.

Reply:
I've resolved this and created [proposal issue #7](/tc39/proposal-eventual-send/issues/7).